client/src/components/Applet/Applet.js

```diff
@@ -1,25 +1,33 @@
-import React, { Component } from "react";
-import FontAwesome from "react-fontawesome";
-import $ from "jquery";
+import React, { Component } from 'react';
+import FontAwesome from 'react-fontawesome';
+import $ from 'jquery';
+import Summary from './../Summary/Summary.js';
+import Highlights from './../Highlights/Highlights.js';
+// import HighlightItem from '../Highlights/HighlightItem.js';
+import Details from './../Details/Details.js';
+import Sleeping from './../Sleeping Arrangments/Sleeping.js';
+import Rules from './../Rules/Rules.js';
+import Cancelation from './../Cancellations/Cancellation.js';
 
 class Applet extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      hello: "world"
+      room: {}
     };
   }
 
   // componentWillMount(){}
   componentDidMount() {
-    const id = parseInt(window.location.pathname.split("/").pop());
+    const id = parseInt(window.location.pathname.split('/').pop());
     const self = this;
     $.get(`/api/rooms/${id}`, function(data) {
-      console.log("success got data", data);
```

Comment:
Consider using fetch instead of jQuery- https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch